### PR TITLE
[FIX] Give the db name to the request's session

### PR DESCRIPTION
**Pull request by adrienpeiffer**
Related to https://github.com/OCA/queue/pull/1

### CODE REVIEW

File: queue_job/controllers/main.py [hunk 1]
```diff
@@ -73,7 +73,7 @@ def old_runjob(self, db, job_uuid, **kw):
 
     @http.route('/queue_job/runjob', type='http', auth='none')
     def runjob(self, db, job_uuid, **kw):
-
+        http.request.session.db = db
```
Comment (sbidoul) [inline queue_job/controllers/main.py:76]: @adrienpeiffer Why is this needed now and not in 9.0?

Perhaps it's the job runner that should `GET /queue_job/runjob?db=<dbname>` ?

Comment (adrienpeiffer) [reply queue_job/controllers/main.py:76]: As far I know, with no authentification the database isn't set in the session.
